apiserver/src/api/error.rs

```diff
@@ -3,16 +3,9 @@ use models::node::BottlerocketShadowError;
 use actix_web::error::ResponseError;
 use snafu::Snafu;
 
-/// The crate-wide result type.
-pub type Result<T> = std::result::Result<T, Error>;
-
-/// The crate-wide error type.
 #[derive(Debug, Snafu)]
 #[snafu(visibility(pub))]
-pub enum Error {
-    #[snafu(display("Unable to create client: '{}'", source))]
-    ClientCreate { source: kube::Error },
-
+pub enum ApiError {
```

Comment:
Throughout our codebase (including other projects) we tend to use `Error` for all error enums instead of giving them bespoke names.

Reply:
Sounds good! I was following some of the other examples found in brupop:

https://github.com/bottlerocket-os/bottlerocket-update-operator/blob/7f8ac76f338af68902d9559ab9d5ede874fe472e/apiserver/src/auth/error.rs#L7

Maybe these are due for a small refactor?